Kalaha/HumanPlayer.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * Benutzer: Daniel
+ * Datum: 09.10.2017
+ * Zeit: 12:54
+ */
+
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System;
+
+namespace Kalaha
+{
+	/// <summary>
+	/// Description of HumanPlayer.
+	/// </summary>
+	public class HumanPlayer : Player, IChooseMove
+	{
+		private int SlotsToChoose;
+		public int ChooseMove()
+		{
+			
+			Console.WriteLine("Spieler {0} ist dran. Bitte Zug eingeben!\n", this.PlaySide);
+			string input = Console.ReadLine();			
+			
+			while(Int32.Parse(input)<1 || Int32.Parse(input)>SlotsToChoose) {		
+				Console.WriteLine("Deine Wahl muss zwischen {0} und {1} liegen. Versuchen wirs nochmal:", 1,SlotsToChoose);
+				input = Console.ReadLine();
+			}
+			return Int32.Parse(input)-1;	

```

Comment:
Zählst du hier die Feldauswahl runter bis du einfach das höchstmögliche nimmst?

Reply:
Nein, das hat folgende Bewandtnis: ReadLine() liest eine Konsoleneingabe ein, aber als string!
Wir wollen ein int zurückgeben. Int32.Parse(input) konvertiert den string "input" in eine integer-variable. Das "-1" dient lediglich dazu, dass man zahlen von 1-6 eingibt und nicht von 0-5 was auf dauer nervt.

Reply:
okidoki, jetzt habs selbst ich verstanden